docs/annotation-cheat-sheet.md

```diff
@@ -36,12 +36,20 @@ If the term has a primitive type
 <Term Name="StringTerm" Type="Edm.String" />
 ```
 
-The annotation value is provided in a [corresponding attribute](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ConstantExpression).
+A constant annotation value is provided in a [corresponding attribute](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ConstantExpression).
 
 ```xml
 <Annotation Term="Vocab.StringTerm" String="annotation value" />
 ```
 
+A dynamic annotation value can be provided for the same term using a [value path expression](https://docs.oasis-open.org/odata/odata-csdl-xml/v4.01/odata-csdl-xml-v4.01.html#sec_ValuePath).
+
+```xml
+<Annotation Term="Vocab.StringTerm" Path="SomeStringProperty" />
+```
+
+The property referenced via the value path expression needs to have the same type as the term. A value path expression can always be used instead of a constant value, also in the more complicated cases below.
+
```

Comment:
Do you also want to mention other dynamic expressions, e.g., edm:Apply?

Reply:
@HeikoTheissen I‘d rather make that a separate page on „Dynamic Expressions“. Do you want to open an issue for it, or/and create a starter pull request for it?

Reply:
Good. I leave it up to you to create another pull request.

Reply:
The OData TC welcomes contributions, so don't be shy 😄

Reply:
Issue created: https://github.com/oasis-tcs/odata-vocabularies/issues/85